translationCoreApps/VerseCheck
# all selection helpers tests pass

DO NOT MERGE, JUST REVIEW :D

## This PR Addresses: 

- Added tests and made all tests pass for `selectionHelpers` in VerseCheck.
- This does not include UI/browser text selection functions yet.

## To Test:

- cd into tC_apps/VerseCheck
- `npm install` to setup the tool to run tests
- `npm test` to run the tests
- ensure all tests pass
- review tests to see if new ones should be added.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/versecheck/70)
